### PR TITLE
Add business date customer document

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Ref:
- Write new documentation on transaction idempotency and business and effective date configuration
- Business & Effective Date Separation - https://github.com/apache/fineract/pull/2358/files
- https://mifosforge.jira.com/wiki/spaces/P/pages/3079733251/Introducing+Business+Date+into+Fineract+community+version
- https://github.com/openMF/mifos-documentation/issues/14

cc @edcable. 

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by galovics** [comment]
@tapaswenipathak are there any plans to make this PR ready? Do you need help with it?

**Comment by tapaswenipathak** [comment]
plans: yes. soon, this week most probably, a couple of more or a bunch.

cc @edcable.

**Comment by vidakovic** [comment]
@tapaswenipathak would be great if you could format the PR title according to this pattern: "FINERACT-XXXX: Title of Jira ticket". If there's no Jira ticket yet and you are unable to create one yourself then please contact me, I'll help out. Thanks.

**Comment by tapaswenipathak** [comment]
sure, but i dont't think any jira ticket(s) exist, can you sync w/ Ed on
slack? please loop me in, if there is anything i can help w/.

**Comment by tapaswenipathak** [comment]
looped in both of you in the email. not sure if there should be one,
multiple or no jira ticket.

File: fineract-doc/src/docs/en/chapters/documentation/business-date.adoc [hunk 1]
```diff
@@ -0,0 +1,150 @@
+= Introducing Business Date into Fineract
```
Comment (galovics) [inline fineract-doc/src/docs/en/chapters/documentation/business-date.adoc:1]: Already present. https://fineract.apache.org/docs/current/#_introducing_business_date_into_fineract_community_version

**Review by galovics — CHANGES_REQUESTED** [review]

File: fineract-doc/src/docs/en/chapters/documentation/business-date.adoc (hunk 1 above)
Comment (tapaswenipathak) [reply fineract-doc/src/docs/en/chapters/documentation/business-date.adoc:1]: cc @edcable.

**Pull request closed without merging**